Purpose: Increase number of allowed dependency checking loops in makeboot.c.
         999 seems to be too low, which causes make like boot to use an
	 incorrect sequence.
Fixes:   None.
Authour: Kel Modderman
Status:  Applied upstream.
---
--- a/makeboot.c
+++ b/makeboot.c
@@ -117,8 +117,11 @@ static int check_loop(struct makenode *d
 			fprintf(stderr, "loop exists %s in %s!\n", dep->name, src->name);
 			return 1;
 		}
-		if (loop++ > 999)
+		if (loop++ > 99999) {
+			fprintf(stderr, "too many loops! (loop=%d, dep->name=%s, src->name=%s)\n",
+				loop, dep->name, src->name);
 			return 1;
+		}
 		if (check_loop(s->node, src))
 			return 1;
 	}
